var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -156,9 +156,8 @@ class PyTensorflow(Package, CudaPackage):
     depends_on('py-google-pasta@0.1.6:', type=('build', 'run'), when='@1.14:2.0')
     depends_on('py-google-pasta@0.1.2:', type=('build', 'run'), when='@1.12.1')
     # propagate the mpi variant setting for h5py/hdf5 to avoid unexpected crashes
-    depends_on('py-h5py@2.10.0:2.10.999', type=('build', 'run'), when='@2.2:+mpi')
```

Comment:
Oops, sorry about that -- I missed the `when='+mpi'` in the first `depends_on`.  The downside of the  updated phrasing is that it duplicates the version restriction.

Reply:
no worries, i missed it too - picked up on it when i went to rebase my tf 2.5 branch